src/proj_api.h

```diff
@@ -192,6 +192,7 @@ void PROJ_DLL pj_cleanup_lock(void);
 
 void PROJ_DLL pj_set_ctx( projPJ, projCtx );
 projCtx PROJ_DLL pj_ctx_alloc(void);
+projCtx PROJ_DLL pj_ctx_clone( projCtx );
```

Comment:
to remove. proj_api.h is legacy and is going to be removed soon

Reply:
:+1: